bin/node/bench/src/common.rs

```diff
@@ -1,6 +1,6 @@
 // This file is part of Substrate.
 
-// Copyright (C) 2020-2022 Parity Technologies (UK) Ltd.
+// Copyright (C) Parity Technologies (UK) Ltd.
```

Comment:
I see 17 about more files with `\d+\s*Parity Technologies`.

Reply:
Tried to fix in https://github.com/paritytech/substrate/pull/13415/commits/8d8117269207f59ef639dd1058a4179c444c8678